### PR TITLE
Switch to bungie name instead of platform name for accounts

### PR DESCRIPTION
**Pull request by bhollis**
image

### CODE REVIEW

**Comment by delphiactual** [comment]
are we zero-padding?
https://github.com/Bungie-net/api/issues/1514

**Comment by bhollis** [comment]
Ah good point I forgot about that

File: src/app/accounts/destiny-account.ts [hunk 1]
```diff
@@ -113,6 +113,14 @@ function couldBeD1Account(destinyAccount: DestinyProfileUserInfoCard | UserInfoC
   );
 }
 
+function formatBungieName(destinyAccount: DestinyProfileUserInfoCard | UserInfoCard) {
+  return (
+    destinyAccount.bungieGlobalDisplayName +
+    (destinyAccount.bungieGlobalDisplayNameCode &&
```
Comment (nev-r) [inline src/app/accounts/destiny-account.ts:119]: isn't this going to append the string `"undefined"` if it's missing?

Comment (bhollis) [reply src/app/accounts/destiny-account.ts:119]: I suppose it will

**Review by delphiactual — APPROVED** [review]

**Pull request merged**